sympy/utilities/tests/diagnose_imports.py

```diff
@@ -0,0 +1 @@
+../../../bin/diagnose_imports.py
```

Comment:
So this is a symbolic link? I wonder if it can cause problems. Is this run as part of our regular test suite?